src/rmoas.types.ts

```diff
@@ -0,0 +1,67 @@
+import { OpenAPIV3, OpenAPIV3_1 } from 'openapi-types';
+// import Oas from '.';
+
+export type OASDocument = OpenAPIV3.Document | OpenAPIV3_1.Document;
+export type OperationObject = OpenAPIV3.OperationObject | OpenAPIV3_1.OperationObject;
+
+export type HttpMethods =
+  | (OpenAPIV3.HttpMethods | OpenAPIV3_1.HttpMethods)
+  | ('get' | 'put' | 'post' | 'delete' | 'options' | 'head' | 'patch' | 'trace');
```

Comment:
Had to add these so we could use them as valid methods because the enum from the `*.HttpMethods` types wasn't doing it.